templates/_lib.hcl

```diff
@@ -100,3 +102,40 @@ ephemeral_disk {
     destination = "local/set_pg_password.sh"
   }
 {%- endmacro %}
+
+{% macro promtail() %}
+    task "promtail" {
+      driver = "docker"
+
+      config {
+        image = "grafana/promtail:master"
```

Comment:
Could we pin this with a digest?

Reply:
We could, I'll add an issue because it needs testing and I don't want to delay the deployment.
https://github.com/liquidinvestigations/node/issues/198